modules/sdk-core/src/bitgo/wallet/iWallets.ts

```diff
@@ -107,6 +107,21 @@ export interface AcceptShareOptions {
   newWalletPassphrase?: string;
 }
 
+export interface BulkAcceptShareOptions {
+  walletShareIds: string[];
+  userPassword: string;
+  newWalletPassphrase?: string;
+}
+
+export interface BulkAcceptShareOptionsRequest {
+  keysForWalletShares: AcceptShareOptionsRequest[];
+}
```

Comment:
There is no reason for this type, `AcceptShareOptionsRequest` can directly be used as an array